soccer/src/soccer/planning/planner/plan_request.cpp

```diff
@@ -27,7 +27,7 @@ void fill_obstacles(const PlanRequest& in, rj_geometry::ShapeSet* out_static,
 
     // Add their robots as static obstacles (inflated based on velocity).
     // See calc_static_robot_obs() docstring for more info.
-    for (int shell = 0; shell < kNumShells; shell++) {
+    for (unsigned long shell = 0; shell < kNumShells; shell++) {
```

Comment:
kNumShells is actually of type size_t. This resolves to unsigned long, so this change removes the compiler warnings, but it would be better if it was something like 
```suggestion
    for (size_t shell = 0; shell < kNumShells; shell++) {
```

Reply:
(if that re-triggers compiler warnings let me know)